__tests__/app.test.js

```diff
@@ -65,4 +65,40 @@ describe('/api/categories', () => {
             })
         })
     })
+
+    describe('PATCH requests', () => {
+        describe('PATCH /api/reviews/:review_id', () => {
+            test('Happy path', () => {
+                const patchSend = {inc_votes: 10}
+                
+                return request(app).patch('/api/reviews/2')
+                .expect(200)
+                .send(patchSend)
+                .then(({body}) => {
+                    expect(body).toEqual({
+                        review_id: 2,
+                        title: 'Jenga',
+                        designer: 'Leslie Scott',
+                        owner: 'philippaclaire9',
+                        review_img_url:
+                          'https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png',
+                        review_body: 'Fiddly fun for all the family',
+                        category: 'dexterity',
+                        created_at: "2021-01-18T10:01:41.251Z",
+                        votes: 15
+                      }) 
+                })
+            })
+            test('Body missing inc_votes', () => {
+                return request(app).patch('/api/reviews/2').expect(400).send({beans: "Oo yes please"}).then(({body}) => {
+                    expect(body.msg).toBe("Patch body must contain the number of incoming votes.")
+                })
+            })
+            test('inc_votes wrong data type', () => {
+                return request(app).patch('/api/reviews/2').expect(400).send({inc_votes: "eleven"}).then(({body}) => {
+                    expect(body.msg).toBe("Input of incorrect data type.")
+                })
+            })
+        })
+    })
```

Comment:
Can i have a test to see what happens if we try to patch an id that doesnt exist and an incorrect data type of review_id